docs/intro.rst

```diff
@@ -3,12 +3,13 @@
 Kafka REST Proxy
 ================
 
-The Kafka REST Proxy provides a RESTful interface to a Kafka cluster. It makes
-it easy to produce and consume messages, view the state of the cluster, and
-perform administrative actions without using the native Kafka protocol or
-clients. Examples of use cases include reporting data to Kafka from any
-frontend app built in any language, ingesting messages into a stream processing
-framework that doesn't yet support Kafka, and scripting administrative actions.
+The Kafka REST Proxy is part of `Confluent Open Source <https://www.confluent.io/product/confluent-open-source/>`_ and `Confluent Enterprise <https://www.confluent.io/product/confluent-enterprise/>`_ distributions. The Proxy provides a RESTful interface to a Kafka cluster, making it easy to produce and consume messages, view the state of the cluster, and perform administrative actions without using the native Kafka protocol or clients.
```

Comment:
`The Proxy` is a bit weird -- should probably be either lowercase `proxy` or the full name `REST Proxy`.

Reply:
fixed https://github.com/confluentinc/kafka-rest/pull/339/commits/af1f060b20bfe8a62d48ef354d1b9a36532d1697